packages/atomic/src/components/atomic-result/atomic-result.tsx

```diff
@@ -146,6 +156,9 @@ export class AtomicResult {
   }
 
   public componentDidLoad() {
+    if (!this.store.get('firstResultLoaded')) {
+      this.store.set('firstResultLoaded', true);
```

Comment:
The first result to ever load sets the flag, this and related use of the flag fixes the placeholder